### PR TITLE
Add @throws docs.

### PR DESCRIPTION
**Pull request by dongjoon-hyun**
Signed-off-by: Dongjoon Hyun <dongjoon@apache.org>

### CODE REVIEW

File: src/pocketmine/level/generator/Generator.php [hunk 1]
```diff
@@ -171,6 +176,9 @@ public static function getFastNoise2D(Noise $noise, $xSize, $zSize, $samplingRat
 	 * @param int   $z
 	 *
 	 * @return \SplFixedArray
+	 *
+	 * @throws \InvalidArgumentException
+	 * @thorws \InvalidArgumentCountException
```
Comment (PocketKiller) [inline src/pocketmine/level/generator/Generator.php]: typo.

Comment (dongjoon-hyun) [inline src/pocketmine/level/generator/Generator.php]: Oops. Sorry! I'll fix it.

**Comment by dongjoon-hyun** [comment]
Thank you for your reviewing, @PocketKiller !

**Comment by dongjoon-hyun** [comment]
I squashed the commits into one.

**Comment by dongjoon-hyun** [comment]
Thank you for merging, @PEMapModder !

**Comment by PEMapModder** [comment]
@dongjoon-hyun it would be nice if you have added description though.

**Comment by dongjoon-hyun** [comment]
I see. Next time, I will add more descriptions. Thank you for your advice.

**Pull request merged**